kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/shared/model/GwtWifiSecurity.java

```diff
@@ -19,7 +19,9 @@ public enum GwtWifiSecurity {
     netWifiSecurityWPA("WPA"),
     netWifiSecurityWPA2("WPA2"),
     netWifiSecurityWPA2WPA3Enterprise("WPA2/WPA3-Enterprise"),
```

Comment:
Probably out of scope for this PR but we should probably not call this WPA2/WPA3 enterprise....

```suggestion
    netWifiSecurityWPA2Enterprise("WPA2-Enterprise"),
```

Reply:
Yeah... but this will break a lot of things... So, I decided not to do this modification for now.